### PR TITLE
Added BestReplyMessages, formatting

### PR DESCRIPTION
**Pull request by Sameer-MSFT**
Added BestReplyMessages, formatting. Please publish on July 1

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: yammer/manage-security-and-compliance/gdpr-requests-in-yammer-basic.md [hunk 1]
```diff
@@ -73,11 +73,13 @@ You can export your own data in Yammer Basic, and can choose to delete any data
...
     
     - **Groups.csv** For any group you created or modified during the specified date range, lists the Yammer ID, name, description, privacy status, whether the group is internal or external, link to the group, creation date, and updated date. This file also includes an aggregated total of the number of polls you voted, and what polls you created. 
-- 
-    -|**LikedMessages.csv** For any message liked by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the ids for attachments, and creation and deletion information. A list of polls you created will also be provided. For announcements, includes the title of the announcement.     
+    
+    - **LikedMessages.csv** For any message liked by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the ids for attachments, and creation and deletion information. A list of polls you created will also be provided. For announcements, includes the title of the announcement.
```
Comment (nenonix) [inline yammer/manage-security-and-compliance/gdpr-requests-in-yammer-basic.md]:
```suggestion
    - **LikedMessages.csv** For any message liked by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the IDs for attachments, and creation and deletion information. A list of polls you created will also be provided. For announcements, includes the title of the announcement.     
```

File: yammer/manage-security-and-compliance/gdpr-requests-in-yammer-basic.md [hunk 2]
```diff
@@ -73,11 +73,13 @@ You can export your own data in Yammer Basic, and can choose to delete any data
...
+    
+    - **LikedMessages.csv** For any message liked by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the ids for attachments, and creation and deletion information. A list of polls you created will also be provided. For announcements, includes the title of the announcement.     
 
     - **Messages.csv** For any message you sent or modified during the specified date range, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the full text of the message, attachments, and creation and deletion information. For announcements, includes the title of the announcement.
     
+   - **BestReplyMessages.csv** For any message marked as best reply by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the ids for attachments, and creation and deletion information.
```
Comment (nenonix) [inline yammer/manage-security-and-compliance/gdpr-requests-in-yammer-basic.md]:
```suggestion
   - **BestReplyMessages.csv** For any message marked as best reply by the user, lists the message ID, thread ID, group ID, group name, privacy status, sender ID, name and email, the full body of the message, the IDs for attachments, creation and deletion information.
```

**Comment by mypil** [comment]
@Sameer-MSFT - we need your kind assistance to merge the new commit for the copy/edit review made by nenonix or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by Sameer-MSFT** [comment]
Looks good, thank you!

**Pull request merged**